src/index.coffee

```diff
@@ -11,6 +11,6 @@ module.exports = (app, options) ->
   if !options or options.enabled isnt false
     autoMigrate = (options and options.migration) or 'auto-update'
     autoMigrate = require './' + autoMigrate
-    autoMigrate(app, options)
+    app.autoMigrateDone = autoMigrate(app, options).then -> debug "Done"
```

Comment:
Yeah, That's a good idea. but `loaded` is not exact. 

* loaded: the component is ready, but not executed.
* done: the component is executed and no error.
* failed: the component is executed, but failed.. 

```coffee
app.set('loopback-component-auto-migrate-status', 'loaded')
autoMigrate(app, options).onCallback (err)->
  if err
    app.set('loopback-component-auto-migrate-error', err)
    app.set('loopback-component-auto-migrate-status', 'failed')
  else
    app.set('loopback-component-auto-migrate-status', 'done')
```